dbbackup/management/commands/dbbackup.py

```diff
@@ -11,7 +11,7 @@
 
 
 class Command(BaseDbBackupCommand):
-    help = "Backup a database, encrypt and/or compress and write to " "storage." ""
+    help = "Backup a database, encrypt and/or compress and write to 'storage.'"
```

Comment:
to be honest there's no reason for `'storage'` to be in quotes here. Let's just remove that.

Reply:
is it better rn? @Archmonger